hsmetrics.wdl

```diff
@@ -0,0 +1,160 @@
+version 1.0
+
+workflow hsMetrics {
+input {
+    File    inputBam
+    File    baitBed
+    File    targetBed
+    String  refFasta
+    String? outputPrefix = ""
+    String? stringencyFilter = "LENIENT"
+    Float?  minPct      = 0.5
+    Int?    coverageCap = 500
+}
+
+call makeRefDictionary { input: refFasta = refFasta }
+call bedToIntervals as bedToTargetIntervals { input: inputBed = targetBed, refDict = makeRefDictionary.refDict }
+call bedToIntervals as bedToBaitIntervals { input: inputBed = baitBed, refDict = makeRefDictionary.refDict }
+
+call collectHSmetrics{ input: inputBam = inputBam, baitIntervals = bedToBaitIntervals.outputIntervals, targetIntervals = bedToTargetIntervals.outputIntervals, refFasta = refFasta, filter = stringencyFilter, coverageCap = coverageCap, outputPrefix = outputPrefix }
+call collectInsertMetrics{ input: inputBam = inputBam, minPct = minPct, outputPrefix = outputPrefix }
+
+output {
+  File outputHSMetrics  = collectHSmetrics.outputHSMetrics
+  File outputINSMetrics = collectInsertMetrics.outputINSMetrics
+  File outputINSPDF     = collectInsertMetrics.outputINSPDF
+}
+
+}
+
+# ==========================================
+#  TASK 1 of 4: make reference dictionary
+# ==========================================
+task makeRefDictionary {
```

Comment:
This should be moved to `modulator`

Reply:
Version 1.1 ?

Reply:
Actually, I have this fixed in a good way - modules for hg19 and hg38 with .dict files are currently awaiting Bob's approval and merge